### PR TITLE
Rewards Claim Data (json) Instructions

### PR DESCRIPTION
**Pull request by wesfloyd**
image

### CODE REVIEW

**Review by seanmcgary — CHANGES_REQUESTED** [review]

File: docs/eigenlayer/rewards-claiming/rewards-snapshot-data.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+---
+sidebar_position: 3
+
+---
+
+# Rewards Distribution Data
+
+Daily rewards snapshot distribution data is available via a public S3 bucket. Users may access this data for their own analytics purposes. Follow these steps in order access the data:
```
[the remark this answers is not in the document]
Comment (wesfloyd) [reply docs/eigenlayer/rewards-claiming/rewards-snapshot-data.md:8]: Hi @seanmcgary can you confirm the comment above
>Mainnet is actually weekly?

The docs [here](https://github.com/Layr-Labs/eigenlayer-contracts/blob/dev/docs/core/RewardsCoordinator.md#off-chain-calculation) from @8sunyuan and @ypatil12 indicate:
>The pipeline takes snapshots of core contract state at the SNAPSHOT_CADENCE, currently set to **once per day**.

Should we update both docs to indicate something like the following?
>Rewards snapshot frequency is once per week on Mainnet and daily on Testnet.

File: docs/eigenlayer/rewards-claiming/rewards-snapshot-data.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+* bucket_url: 
+  * [https://eigenlabs-rewards-testnet-holesky.s3.amazonaws.com](https://eigenlabs-rewards-testnet-holesky.s3.amazonaws.com)
+  * [https://eigenlabs-rewards-mainnet-ethereum.s3.amazonaws.com](https://eigenlabs-rewards-mainnet-ethereum.s3.amazonaws.com)
+* environment: testnet or mainnet
+* network: holesky or ethereum
+
+Example:
+
+`https://eigenlabs-rewards-testnet-holesky.s3.amazonaws.com/testnet/holesky/1723075200/claim-amounts.json`
```
[the remark this answers is not in the document]
Comment (wesfloyd) [reply docs/eigenlayer/rewards-claiming/rewards-snapshot-data.md:34]: Good call, thanks. Updating now

File: docs/eigenlayer/rewards-claiming/rewards-snapshot-data.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (wesfloyd) [reply docs/eigenlayer/rewards-claiming/rewards-snapshot-data.md:8]: good call - adjusted the language now

File: docs/eigenlayer/rewards-claiming/rewards-snapshot-data.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+* Get the DistributionRoot(s) needed for the rewards time ranges desired. 
+  * Call `getCurrentDistributionRoot` which will give you the most recent root posted. `getCurrentClaimableDistributionRoot` will give you the most recent claimable root since there is an activation delay.
+    * You may also call `getCurrentDistributionRoot` which will give you the most recent root posted. `getCurrentClaimableDistributionRoot` will give you the most recent claimable root since there is an activation delay.
+  * Find the rewardsCalculationEndTimestamp value as the second value in the [DistributionRoot struct](https://github.com/Layr-Labs/eigenlayer-contracts/blob/b4fa900a11df04f3b0034e225deb1eb42b39f8bc/src/contracts/interfaces/IRewardsCoordinator.sol#L72) resulting tuple.
+  * Or Index on the event `DistributionRootSubmitted` which is emitted when a [root is created](https://etherscan.io/tx/0x2aff6f7b0132092c05c8f6f41a5e5eeeb208aa0d95ebcc9022d7823e343dd012#eventlog).
+  * Note: the current snapshot cadence is once per day ([more detail here](https://github.com/Layr-Labs/eigenlayer-contracts/blob/dev/docs/core/RewardsCoordinator.md#off-chain-calculation)).
```
[the remark this answers is not in the document]
Comment (wesfloyd) [reply docs/eigenlayer/rewards-claiming/rewards-snapshot-data.md:19]: thanks, updated accordingly

**Review by dabit3 — APPROVED** [review]

**Review by seanmcgary — APPROVED** [review]

**Pull request merged**